css-inline/src/html/attributes.rs

```diff
@@ -1,12 +1,79 @@
 use html5ever::{local_name, namespace_url, ns, tendril::StrTendril, QualName};
-use std::collections::BTreeMap;
+use rustc_hash::FxHasher;
+use selectors::{
+    attr::CaseSensitivity,
+    bloom::{BloomStorageBool, CountingBloomFilter},
+};
+use std::{
+    collections::BTreeMap,
+    hash::{Hash, Hasher},
+};
+
+#[derive(Debug)]
+pub(crate) struct Class {
+    pub(crate) value: StrTendril,
+    pub(crate) cache: CountingBloomFilter<BloomStorageBool>,
```

Comment:
Need a comment explaining why the Bloom filter is here and the observation from real use cases which make its use worthwhile